app/src/components/market/sections/market_create/steps/items/import_market_content.tsx

```diff
@@ -82,11 +82,16 @@ const CommentLabel = styled.div`
   }
 `
 
-const TitleValueVertical = styled(TitleValue)`
+const TitleValueVertical = styled(TitleValue)<{ verified?: boolean }>`
   flex-direction: column;
   justify-content: flex-start;
   text-transform: capitalize;
 
+  &:last-child {
+    p {
+      ${({ verified }) => (verified ? '' : `margin-top:-3px;`)};
```

Comment:
Why is `margin-top: -3px` necessary?

Reply:
@liaojikunwork well because in the difference in size of buttons that issue you mentioned on keybase with icon not vertically aligned should be fixed with this...

Reply:
I know, but it's not a professional way to fix this issue.
Please export alert svg on Figma again( it will be 17*17) and replace svg file.
On mobile, icon size is different, so just setting `margin-top: -3px` is not a good way, I think.